.clang-tidy

```diff
@@ -20,6 +20,7 @@ Checks: >
   -abseil-string-find-str-contains,
   -bugprone-easily-swappable-parameters,
   -bugprone-unchecked-optional-access,
+  -bugprone-unused-return-value,
```

Comment:
Are we sure that we want to disable this check? Actually, it sounds useful, but I don't know how useful it actually is. How many methods return useless return values?

Reply:
Yes it is, the problem is that assignment operators lead to false positives (they didn't before)

Reply:
One would need to rewrite
`action = json.get<std::unique_ptr<Mutations<Nucleotide>>>();`
to
`(void) (action = json.get<std::unique_ptr<Mutations<Nucleotide>>>());`
to avoid this error when not using an assignment further...

Reply:
I add a description

Reply:
Ok, the assignment thing is ugly. Then let's have it disabled :+1: